src/TwigComponent/src/ComponentFactory.php

```diff
@@ -30,7 +30,7 @@ final class ComponentFactory
      * @param array<class-string, string> $classMap
      */
     public function __construct(
-        private ComponentTemplateFinderInterface $componentTemplateFinder,
+        private ComponentTemplateFinder $componentTemplateFinder,
```

Comment:
Should still be the interface, right? As we can't remove the interface

Reply:
Hmm.. i'll suggest an other solution then :)